kreport2mpa.py

```diff
@@ -140,7 +148,7 @@ def main():
     #Read through report file 
     main_lvls = ['R','K','D','P','C','O','F','G','S']
     for line in r_file:
-        report_vals = process_kraken_report(line)
+        report_vals = process_kraken_report(line, args.remove_spaces)
```

Comment:
```suggestion
        report_vals = process_kraken_report(line, args.foobar)
```
Is this a better name